File: source/shared/eval/forward-fixtures/9.1.0/codebase/candidate-dotnet/CheckoutController.cs
public sealed class CheckoutController
{
    public Response CreateOrder(Request request)
    {
        try
        {
            var order = Order.From(request);
            repository.Save(order);
            return Response.Json(201, new { status = "created" });
        }
        catch (ArgumentException)
        {
            return Response.Json(500, new { error = "INTERNAL_ERROR" });
        }
    }

    public Response CancelOrder(string id)
    {
        var order = repository.Get(id);
        repository.Delete(order);
        return Response.Json(204, null);
    }
}

File: source/shared/eval/forward-fixtures/9.1.0/codebase/legacy-qt/checkout_controller.cpp
#include <string>

Response CheckoutController::createOrder(const Request& request) {
    if (request.amountCents <= 0) {
        return Response{400, "{\"error\":\"INVALID_AMOUNT\"}"};
    }
    repository.save(Order::from(request));
    return Response{201, "{\"status\":\"created\"}"};
}

Response CheckoutController::cancelOrder(const std::string& id) {
    auto order = repository.get(id);
    order.cancel();
    repository.save(order);
    return Response{200, "{\"status\":\"cancelled\"}"};
}
